sb.web.app/src/App.vue

```diff
@@ -141,7 +141,7 @@ export default {
 
     setInterval(async () => {
       this.setStoreData()
-    }, 200000)
```

Comment:
This was accidentally merged with my last pr it is a revert back to the normal state